pkg/bootstrap/bootstrap_config.go

```diff
@@ -254,3 +259,93 @@ func WriteBootstrap(config *meshconfig.ProxyConfig, node string, epoch int, pilo
 	err = t.Execute(fout, opts)
 	return fname, err
 }
+
+// CheckinData has data provided by Pilot needed to start and prepare envoy.
+type CheckinData struct {
+	AvailabilityZone string
+}
+
+var (
+	// IstioCertDir is the location of istio certificates. Visible for testing.
+	// Can be set using ISTIO_CERT_DIR environment variable.
+	IstioCertDir = "/etc/certs"
+
+	// PilotSAN is the DNS name included in the pilot certificate. It may not match the
+	// actual address. Pilot, IstioCA and Sidecar have special certificate that include both
+	// SPIFEE and DNS.
+	PilotSAN = "istio-pilot.istio-system.svc"
```

Comment:
I assume this is not going to work in environments not named Kube? That was my assumption when I wrote it originally. We might have to do some work later to get it to work with other SRs.

Reply:
Could you stick a TODO there so its a bit more obvious that it may be an issue down the line?

Reply:
Why wouldn't it work ? That's the name in the cert - it should work anywhere.  

Maybe a better option is to fix the ca to not use '.svc' in the name, it may be confusing.